e2e/basic/main_test.go

```diff
@@ -2,19 +2,29 @@ package basic
 
 import (
 	"os"
-	"testing"
-
-	"github.com/sirupsen/logrus"
-
-	"github.com/celestiaorg/knuu/pkg/knuu"
 )
 
-func TestMain(m *testing.M) {
-	err := knuu.Initialize()
-	if err != nil {
-		logrus.Fatalf("error initializing knuu: %v", err)
+func convertViaMap(b bool) int {
+	table := map[bool]int{
+		true:  1,
+		false: 0,
 	}
-	logrus.Infof("Scope: %s", knuu.Scope())
-	exitVal := m.Run()
-	os.Exit(exitVal)
+	return table[b]
+}
+
+func (s *TestSuite) TestMain() {
+	s.T().Parallel()
+	// Setup
+
+	// Test Logic
+	s.T().Log("Running test case: TestMain")
+
+	// Perform the test
+	exitVal := s.Run("TestMain", func() {
+		s.T().Logf("Scope: %s", s.Knuu.Scope())
+	})
+
+	exitValue := convertViaMap(exitVal)
+
+	os.Exit(exitValue)
 }
```

Comment:
This testmain func is not really needed. if we convert all our basic tests to test suit, this func can be removed

Reply:
will be removed in another PR